lib/src/source_visitor.dart

```diff
@@ -3023,8 +3043,8 @@ class SourceVisitor extends ThrowingAstVisitor {
   /// and followed by the given functions.
   void visitNodes(Iterable<AstNode> nodes,
       {void Function() before,
-      void Function() between,
-      void Function() after}) {
+        void Function() between,
```

Comment:
When did this change?

Reply:
Not sure, may just be from my copy/pasting some code and forgetting to format it. Reformatted the whole file.